profile_library/tuya/TY-A60-15W/model.json

```diff
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-01-23T01:54:09.746937",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "TP-Link HS110",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.2:docker"
+  },
+  "name": "TY-A60-15W E27",
```

Comment:
Could you use the marketed name here.
For example `Tuya Wifi Bulb E27 RGBWW 15W`